### PR TITLE
add basic build instructions

### PR DESCRIPTION
**Pull request by MattAtkinson**

### CODE REVIEW

**Comment by abhijit-mahajani-imgtec** [comment]
We already have build instructions for our mikro-e port at https://github.com/CreatorDev/contiki/blob/3.0-mikro-e/platform/mikro-e/Readme.md
Please do not add anything specific to CreatorKit in generic contiki readme. It will not get accepted while upstreaming.

File: README.md [hunk 1]
```diff
@@ -17,3 +17,20 @@ and so on.
 For more information, see the Contiki website:
 
 [http://contiki-os.org](http://contiki-os.org)
+
+=====
+
+## 6LoWPAN Clicker Build Instructions
+
+**If you would like to build the IoT Creator Kit projects, go to https://github.com/CreatorKit/build and follow the instructions there.**
```
Comment (abhijit-mahajani-imgtec) [inline README.md:25]: contiki build instructions and build makefile has different command line arguments. so let's not confuse people by pointing to CreatorKit/build from here.

**Comment by MattAtkinson** [comment]
No one coming to this repository is going to have any idea how to start with Contiki on 6LowPAN Clicker. None of the 3(!) top level readme files help, neither does the examples folder. How will users know that the Clicker is in platform/mikro-e? mikro-e is not written anywhere on the board silkscreen or box. I'll put all this information into the main docs and hope people don't come to this repo looking to start with Contiki.

**Comment by nikhil-zinjurde-imgtec** [comment]
test with jenkins - dev

**Pull request closed without merging**